src/styles/blocks/catalog.scss

```
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $animation-duration;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
+    }
```

```suggestion
    transition: $animation-duration;

    &:hover {
      transform: scale(1.2);
      color: #34568b;
    }
```
```suggestion
    transition: transform $animation-duration, color $animation-duration;

    &:hover {
      transform: scale(1.2);
      color: #34568b;
    }
```
you have to set transition only for color and transform